pyproject.toml

```diff
@@ -45,8 +45,9 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.9"
-pydantic-settings = "^2.0.2"
 sqlalchemy-helpers = ">=0.12.1"
+flask = "^3.0.3"
+toml = "^0.10.2"
```

Comment:
I think the recommended module for TOML is `tomli`, that's what was imported in Python standard lib in 3.11.